metadata/gc.go

```diff
@@ -41,6 +41,55 @@ func scanRoots(ctx context.Context, tx *bolt.Tx, nc chan<- gc.Node) error {
 		nbkt := v1bkt.Bucket(k)
 		ns := string(k)
 
+		lbkt := nbkt.Bucket(bucketKeyObjectLeases)
```

Comment:
L23:
```go
labelGCRoot       = []byte("containerd.io/gc.root")
```

Do we need to keep `gc.root`?

Reply:
Yes, we will continue to support the `gc.root` label, it just doesn't need to be part of the normal client flows. Client's can still use these labels to retain content not otherwise referenced